tox.ini

```diff
@@ -38,11 +40,14 @@ setenv =
     unit: SNOWFLAKE_TEST_TYPE = unit
     integ: SNOWFLAKE_TEST_TYPE = integ
     doctest: SNOWFLAKE_TEST_TYPE = doctest
-    parallel: SNOWFLAKE_PYTEST_OPTS = {env:SNOWFLAKE_PYTEST_OPTS:} -n auto
     # Add common parts into pytest command
     SNOWFLAKE_PYTEST_COV_LOCATION = {env:JUNIT_REPORT_DIR:{toxworkdir}}/junit.{envname}-{env:cloud_provider:dev}.xml
-    SNOWFLAKE_PYTEST_COV_CMD = --cov snowflake.snowpark --cov-context=test --junitxml {env:SNOWFLAKE_PYTEST_COV_LOCATION} --cov-report=
-    SNOWFLAKE_PYTEST_CMD = pytest {env:SNOWFLAKE_PYTEST_OPTS:} {env:SNOWFLAKE_PYTEST_COV_CMD}
+    SNOWFLAKE_PYTEST_COV_CMD = --cov snowflake.snowpark --junitxml {env:SNOWFLAKE_PYTEST_COV_LOCATION} --cov-report=
+    doctest: SNOWFLAKE_PYTEST_CMD = pytest {env:SNOWFLAKE_PYTEST_OPTS:} {env:SNOWFLAKE_PYTEST_COV_CMD}
```

Comment:
`-n auto` should be specified in `SNOWFLAKE_PYTEST_OPTS` instead of in `SNOWFLAKE_PYTEST_CMD`